api_v3/lib/types/filters/KalturaUserAppRoleFilter.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * @package api
+ * @subpackage filters
+ */
+class KalturaUserAppRoleFilter extends KalturaUserAppRoleBaseFilter
+{
+	/**
+	 * The User Id to search for
+	 *
+	 * @var string
+	 */
+	public $userIdEqual;
+	
+	/**
+	 * Users Ids csv list
+	 *
+	 * @var string
+	 */
+	public $userIdIn;
+	
+	
+	static private $map_between_objects = array
+	(
+		"userIdEqual" => "_eq_user_id",
+		"userIdIn" => "_in_user_id",
+	);
+	
+	public function getMapBetweenObjects()
+	{
+		return array_merge(parent::getMapBetweenObjects(), self::$map_between_objects);
+	}
+	
+	protected function getCoreFilter()
+	{
+		return new AppRoleFilter();
+	}
+	
+	/**
+	 * @param KalturaFilterPager $pager
+	 * @param KalturaDetachedResponseProfile|null $responseProfile
+	 * @return KalturaListResponse
+	 * @throws PropelException
+	 */
+	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
+	{
+		$response = new KalturaUserAppRoleListResponse();
+		
+		if (in_array(kCurrentContext::getCurrentSessionType(), array(kSessionBase::SESSION_TYPE_NONE,kSessionBase::SESSION_TYPE_WIDGET)))
```

Comment:
Check if this is needed.
If you enforce this in the permissions for the service it should be blocked